charters/PORTABILITY/charter.md

```diff
@@ -0,0 +1,57 @@
+# Pipeline Portability Service
+
+## Description
+
+The portability service enables the confirmation that pipelines developed for the HCA DCP can be successfully ran outside of the HCA DCP.
+
+## Objectives
+
+The portability service provides the ability to run HCA DCP pipelines 
+
+## In-scope
+
+* Programmatic authorized access to submit pipelines that will be ran to confirm they execute to completion on different computing infrastructure (execution environments)
+* Programmatic authorized access to extend the backend of the portability service to include additional execution environments
+* A UI to describe the current state of submitted pipelines and execution environments
+* Documentation to describe how to submit a backend or pipeline the system
+* Documentation on portability best practices
+
+### Scientific "guardrails"
+
+Portability service will not comment on scientific accuracy of pipelines, with this constraint it will not need guardrails
+
+## Out-of-scope
+
+* Portability service will not comment on scientific accuracy of pipelines
+* Educational training or user assistance on how to create pipelines
+* Updating backends so they support pipeline execution
+* Updating pipelines so they are portable to environments that do not support workflow language standards
```

Comment:
It's odd that _workflow language standards_ are not referenced in the in-scope section. Should this be called out?